File: src/main/java/com/example/hiringhouseapplication/entity/Bill.java
package com.example.hiringhouseapplication.entity;

import jakarta.persistence.*;
import lombok.*;
import org.springframework.data.annotation.CreatedDate;

import java.util.Date;
@Getter
@Setter
@Entity
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Bill {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer billID;
    private int currentNumberElectrics;
    private int currentNumberWater;
    @CreatedDate
    private Date creatAt;
    private long total;

    @ManyToOne
    @JoinColumn(name = "room_id")
    private Room room;
}

File: house_manager/app/src/main/java/com/example/house_manager/Model/Bill.kt
package com.example.house_manager.Model

import java.util.*

data class Bill(
    val billid: Int,
    val created_at: Date,
    val current_number_electrics: Int,
    val current_number_water: Int,
    val number_electric_before: Int,
    val number_water_before: Int,
    val room_id: String?
)
